guides/common/modules/con_postgresql-as-an-external-database-considerations.adoc

```diff
@@ -10,7 +10,12 @@ ifndef::katello,orcharhino,satellite[]
 Foreman uses the PostgreSQL database.
 endif::[]
 If you want to use PostgreSQL as an external database, the following information can help you decide if this option is right for your {Project} configuration.
+ifdef::foreman-deb[]
+{Project} supports PostgreSQL version 13 and later.
```

Comment:
Should we declare support of PostgreSQL 13 and later for all flavours?

Reply:
Do we test any version other than PG13? I am hesitant to claim something in our documentation we do not test and have users use a version of Postgres with breaking changes.

Reply:
Perhaps let's add a note that we've only tested PG13 and not other PG versions?

Reply:
I have reworded it for foreman-deb. Does that work for everyone?